woocommerce/woocommerce
# Marketplace search no-results style update

### Changes proposed in this Pull Request:

This PR is part of https://github.com/woocommerce/woocommerce/pull/40159

The PR updates labels in the NoResults component to support themes and extensions.
It also updates the no results icon for themes to match the design.

We don't have Most popular themes for now, so this PR temporarily hides them on no-results for themes.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. [Search for something that won't produce results](http://localhost:8888/wp-admin/admin.php?page=wc-admin&path=%2Fextensions&tab=search&category=&term=pay3434)
2. See that themes and extensions have different icons in the NoResults section
3. Read the labels and confirm that they correctly say themes/extensions based on the section
4. Confirm that themes don't have a Most popular list



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Update marketplace search no results to support themes.

</details>
